Make codeformat available for toolbar

ArthurRM opened the pull request:
Github issues:
#9733 

Description of Changes:
Register codeformat also as toolbar item as hinted by @TheSpyder in related discussion (https://github.com/tinymce/tinymce/discussions/7966)

Pre-checks:
[ ] Changelog entry added
[ ] Tests have been added (if applicable)
[ ] Branch prefixed with `feature/`, `hotfix/` or `spike/`

Review:
[ ] Milestone set
[ ] Docs ticket created (if applicable)

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->

## Summary by CodeRabbit

* **Security**
  * Fixed multiple stored XSS vulnerabilities in media plugin, protected comments, and attribute handling.
  * Updated DOMPurify dependency to 3.4.5.

* **New Features**
  * Added `content_language` option to set language attributes on editor elements.
  * Added Code format button to toolbar.

* **Bug Fixes**
  * Fixed list toggling edge cases, selection/body deletion issues, and URI handling for non-Latin1 characters.
  * Improved screen reader announcements for alert/confirm dialogs.

* **Improvements**
  * Enhanced inline diff highlight styling.
  * Improved word wrapping for long strings in code blocks.
  * Refined selection styling for diff annotations.

<!-- end of auto-generated comment: release notes by coderabbit.ai -->

ArthurRM committed:
Make codeformat available for toolbar

TheSpyder reviewed (commented):
I thought this looked a bit wrong, but I think it's the best we could expect from a contributor given how the rest of the file is structured.

I'm mostly curious about the wider context. It's not clear if `onSetupStateToggle` is incorrect here, or `onSetupEditableToggle` is incorrect at the bottom where the menu item is registered. I don't expect you to answer that question; it might need one of our expert devs to take over.

ArthurRM commented:
> I thought this looked a bit wrong, but I think it's the best we could expect from a contributor given how the rest of the file is structured.
> 
> I'm mostly curious about the wider context. It's not clear if `onSetupStateToggle` is incorrect here, or `onSetupEditableToggle` is incorrect at the bottom where the menu item is registered. I don't expect you to answer that question; it might need one of our expert devs to take over.

I basically looked at how the bold/italic/underline/... items are registered; and kept this in sync with those. They also use `onSetupEditableToggle` in the menubar and `onSetupStateToggle` in the toolbar.

ltrouton committed:
Merge remote-tracking branch 'origin/main' into release/8

ltrouton committed:
Revert files

zuzanna-antas committed:
TINY-14294: apply inline selection styles to ins/del elements (#11080)

zuzanna-antas committed:
TINY-14188: fix code overflowing the container in the ai html content (#11083)

Related Ticket: TINY-14188

Description of Changes:

- Added `overflow-wrap: break-word` to the AI chat HTML content code block styles to prevent long unbreakable strings from overflowing their container.

Pre-checks:

- [x] ~~Changelog entry added~~
- [x] ~~Tests have been added (if applicable)~~
- [x] Branch prefixed with `feature/`, `hotfix/` or `spike/`

Review:

- [x] Milestone set
- [x] ~~Docs ticket created (if applicable)~~

GitHub issues (if applicable):

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->

## Summary by CodeRabbit

- **Style**
    - Improved code block display in AI chat by enabling word wrapping for long strings.

<!-- end of auto-generated comment: release notes by coderabbit.ai -->

ltrouton committed:
TINY-13649: Create changelog for TinyMCE 8.5 release (#11071)

ltrouton committed:
TINY-13640: Bunp package.json version for next release

spocke committed:
TINY-14333: Fixed stored XSS vulnerability through `data-mce-` prefixed `src`, `href`, `style` attributes (#22)

spocke committed:
TINY-14353: Fixed stored XSS vulnerability through `mce:protected` comments (#21)

spocke committed:
TINY-14357: Fixed data-mce-object injection (#18)

MitchC1999 committed:
TINY-14224: Add changelog for 8.5.1 patch release

MitchC1999 committed:
TINY-14224: Lint fix for 8.5.1 patch release

MitchC1999 committed:
TINY-14224: Fix failing test for 8.5.1 patch release

MitchC1999 committed:
TINY-14224: Bump version for next patch release

spocke committed:
TINY-14430: Updated dompurify to 3.4.5 (#11120)

Related Ticket: TINY-14430

Description of Changes:
* Updated DOMPurify to 3.4.5

Pre-checks:
* [x] Changelog entry added
* [x] ~~Tests have been added (if applicable)~~
* [x] Branch prefixed with `feature/`, `hotfix/` or `spike/`

Review:
* [x] Milestone set
* [x] Docs ticket created (if applicable)

GitHub issues (if applicable):

ltrouton committed:
TINY-14412: Update version for 8.6.0 release

ltrouton committed:
TINY-14412: Stabilise changelog for 8.6 release

ArthurRM committed:
Merge tag '8.6.0'
